.github/actions/bumpVersion/bumpVersion.js

```diff
@@ -34,50 +37,41 @@ function updateNativeVersions(newVersion) {
         });
 }
 
-// Use Github Actions' default environment variables to get repo information
-// https://docs.github.com/en/free-pro-team@latest/actions/reference/environment-variables#default-environment-variables
-const [repoOwner, repoName] = process.env.GITHUB_REPOSITORY.split('/');
-
-// Determine current patch version
-const {version} = JSON.parse(fs.readFileSync('./package.json'));
-const currentPatchVersion = version.split('-')[0];
-console.log('Current patch version:', currentPatchVersion);
-
-let newVersion;
+const octokit = github.getOctokit(core.getInput('GITHUB_TOKEN', {required: true}));
+let semanticVersionLevel = core.getInput('SEMVER_LEVEL', {require: true});
 
-// Fetch tags
+if (!semanticVersionLevel || !_.find(versionUpdater.semanticVersionLevels, v => v === semanticVersionLevel)) {
```

Comment:
You forgot to rename the constants in this file...

```suggestion
if (!semanticVersionLevel || !_.find(versionUpdater.SEMANTIC_VERSION_LEVELS, v => v === semanticVersionLevel)) {
```